Microsoft.SystemForCrossDomainIdentityManagement/Protocol/ProtocolExtensions.cs

```diff
@@ -42,6 +53,36 @@ public static HttpMethod PatchMethod
             }
         }
 
+        public static bool IsResourceType(this object jobject, string scheme)
```

Comment:
"jobject" is not an expressive name, and the declaration as object doesn't tell us anything. Do you maybe mean, "json?"